api/src/uvisor-input.S

```diff
@@ -134,6 +137,9 @@ __uvisor_debug_driver:
     .long 0
     .long 0
 
+__uvisor_public_box_stack_limit:
+    .long __StackLimit
```

Comment:
Why do we need the public box stack limit twice? Why not just put `__StackLimit` where `__uvisor_public_box_stack_limit` is instead?

Reply:
We feel that the var name __StackLimit is confusing while having 4 different stucks, and we wish to keep uvisor-input and linker.h variables the same